### DISCUSSION TITLE
UI Revamp?

### DISCUSSION THREAD

**Discussion by fchu** [borgbase/vorta#2038] [Ideas]
Hi there,

Wanted to reach out to first of all say thank you for building Vorta!
One thing I noticed using Vorta is that the overall UI and mental model is hard to grasp, eg:
- There are many states that are unclear which other concepts they depend on so it's hard to see how changes to the values of those dropdowns affect the rest of the configuration, eg:
  -  `SSH Key` dropdown
  -  `Repository` dropdown
  - `Start Backup` wrt to whic profile is selected
- Creating a repo vs connecting to a repo is also confusing, I would separate the creation entirely from profiles
- a few more UI states that make understanding of what's going on hard

Obviously, my description above is confusing because I myself don't fully understand what was the original intent.
I'd love to help and contribute from an UX perspective, but I'm not sure where to start.

**Comment by m3nu** [comment]
Thanks for your input. Anything we could change without a big rewrite, for which there won't be time?
